src/index.js

```diff
@@ -141,8 +141,8 @@ export default class Kitsu {
    */
   async get (model, params = {}, headers = {}) {
     try {
-      let [ res, id ] = model.split('/')
-      const url = this.plural(this.resCase(res)) + (id ? '/' + id : '')
+      let [ res, id, relationship ] = model.split('/')
+      const url = this.plural(this.resCase(res)) + (id ? '/' + id + (relationship ? '/' + relationship : '') : '')
```

Comment:
Can the relationship logic be split out into a separate variable and appended to the `url` instead - the nested ternary operations make this hard to read.

The relationship will also need to be wrapped in the `plural()` and `resCase()` functions like `res`

Reply:
A relationship isn't always a `hasMany` relationship.
A author of a comment will be loaded with uri: `comments/4345/author` (hasOne relationship)
Comments of a author will be loaded with uri: `authors/123/comments` (hasMany relationship)

See: http://jsonapi.org/format/#fetching-resources